src/main/java/edu/ucsb/cs156/example/documents/RedditPost.java

```diff
@@ -0,0 +1,31 @@
+package edu.ucsb.cs156.example.documents;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import org.springframework.data.annotation.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RedditPost {
+    @Id
+    private String _id;
+
+    private String id;
+    private String subreddit;
+    private String selftext;
+    private String selftext_html;
+    private String title;
+    private String author;
+    private String url;
+    private List<RedditFlair> link_flair_richtext;
```

Comment:
Can we use `@JsonProperty` ([docs](https://github.com/FasterXML/jackson-annotations#annotations-for-renaming-properties)) to rename the fields to match Java code style conventions?

Reply:
I would prefer not to in this example.    I'm trying to emphasize how Jackson works in the default case.

We could do that in a second example (or maybe I should say a third example, since this is the second example in the MongoDB series.)

I learned from the team02 experience that it is important to try to keep sample code simple, and introduce extra cognitive load only after the simple concepts have been absorbed.